cmd/kubermatic-installer/cmd_quickstart.go

```diff
@@ -0,0 +1,377 @@
+	kindKubeConfigPath := filepath.Join(dir, "kube-config.yaml")
+	kindKubeConfig, err := os.Create(kindKubeConfigPath)
+	if err != nil {
+		logger.Fatalf("failed to create 'kind' cluster kubeconfig: %v", err)
+	}
+	kubeconfigCmd.Stdout = kindKubeConfig
+	if err = kubeconfigCmd.Run(); err != nil {
+		logger.Fatalf("failed to write 'kind' cluster kubeconfig: %v", err)
+	}
+	if err := kindKubeConfig.Close(); err != nil {
+		logger.Fatalf("failed to close 'kind' cluster kubeconfig: %v", err)
+	}
+
+	flag.Set("kubeconfig", kindKubeConfigPath)
+	ctrlConfig, err := ctrlruntimeconfig.GetConfig()
+	if err != nil {
+		logger.Fatalf("failed to initialize runtime config: %w", err)
+	}
+
+	mgr, err := manager.New(ctrlConfig, manager.Options{
+		MetricsBindAddress:     "0",
+		HealthProbeBindAddress: "0",
+	})
+	if err != nil {
+		logger.Fatalf("failed to construct mgr: %w", err)
+	}
+
+	// start the manager in its own goroutine
+	appContext := context.Background()
+
+	go func() {
+		if err := mgr.Start(appContext); err != nil {
+			logger.Fatalf("Failed to start Kubernetes client manager: %v", err)
+		}
+	}()
+
+	// wait for caches to be synced
+	mgrSyncCtx, cancel := context.WithTimeout(appContext, 30*time.Second)
+	defer cancel()
+	if synced := mgr.GetCache().WaitForCacheSync(mgrSyncCtx); !synced {
+		logger.Fatal("Timed out while waiting for Kubernetes client caches to synchronize.")
+	}
+	return mgr.GetClient(), cancel
+}
+
+func writeFileFromURL(url, filepath string) error {
+	resp, err := http.Get(url)
+	if err != nil {
+		return fmt.Errorf("Failed to get example file %v: %v", url, err)
+	}
+	if resp.StatusCode != 200 {
+		return fmt.Errorf("Failed to get example file %v: status code %v", url, resp.StatusCode)
+	}
+	defer resp.Body.Close()
+	file, err := os.Create(filepath)
+	if err != nil {
+		return fmt.Errorf("Failed to create file %v: %v", filepath, err)
+	}
+	writer := bufio.NewWriter(file)
+	defer writer.Flush()
+	if _, err := io.Copy(writer, resp.Body); err != nil {
+		return fmt.Errorf("Failed to write example file %v: %v", filepath, err)
+	}
+	return nil
+}
+
+func ensureResource(kubeClient client.Client, logger *logrus.Logger, o client.Object) {
+	if err := kubeClient.Create(context.Background(), o); err != nil && !errors.IsAlreadyExists(err) {
+		logger.Fatal(err)
+	}
+}
+
+func installKubevirt(logger *logrus.Logger, dir string, kubeClient client.Client) {
+	logger.Info("Installing KubeVirt ...")
+	// TODO: package this whole thing in KKP charts
+	ko := "https://github.com/kubevirt/kubevirt/releases/download/v1.0.0-alpha.0/kubevirt-operator.yaml"
+	kcr := "https://github.com/kubevirt/kubevirt/releases/download/v1.0.0-alpha.0/kubevirt-cr.yaml"
+	cdio := "https://github.com/kubevirt/containerized-data-importer/releases/download/v1.56.0/cdi-operator.yaml"
+	cdicr := "https://github.com/kubevirt/containerized-data-importer/releases/download/v1.56.0/cdi-operator.yaml"
```

Comment:
I guess it might be better to bake this in under https://github.com/kubermatic/kubermatic/tree/v2.22.2/charts